timer.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+"""
+Pedestrian Tracking
+2018
+POVa - Computer Vision
+FIT - Faculty of Information Technology
+BUT - Brno University of Technology
+"""
+import timeit
+
+import cv2
+
+import detector
```

Comment:
https://github.com/mvondracek/VUT-FIT-POVa-2018-Pedestrian-Tracking/commit/e2b012d76ec0d51c92e7301cab718ffe5d6ad4e6#commitcomment-31822398

Reply:
The problem is that _utils/timer.py_ imports e.g. _person.py_ that imports e.g. "_from utils import anything_" etc. However, the launched _timer.py_ in _utils_ does not know "_utils_". Maybe there is an easy solution, but without modyfing sys.path or other files, this was the easiest and fastest solution I came up with.

Reply:
Where does this file import `person` module?

---

> However, the launched timer.py in utils does not know "utils". 

What do you mean by that, please?